javacord-api/src/main/java/org/javacord/api/entity/message/Message.java

```diff
@@ -1544,20 +1544,47 @@ default boolean canDelete(User user) {
      * Replies to this message with the given text.
      *
      * @param messageContent The text to reply with.
-     * @return The sent message.
+     * @return The message that was sent.
      */
     default CompletableFuture<Message> reply(String messageContent) {
-        return new MessageBuilder().replyTo(getId()).setContent(messageContent).send(getChannel());
+        return reply(messageContent, true);
+    }
+
+    /**
+     * Replies to this message with the given text.
+     *
+     * @param messageContent The text to reply with.
+     * @param assertReferenceExists If true, tells discord to throw an error if the message reference does not exist,
+     *                             if false, then leaves out the message reference,
+     *                             if it does not exist but still sends the message.
```

Comment:
```suggestion
     * @param assertReferenceExists If true, throw an error if the message you are replying to does not exists otherwise, 
     *                                                   if false send the message regardless without a reference to a message.
```
The same for the other Javadoc